packages/webcomponents/src/components/business-form-stepped/business-form-stepped.tsx

```diff
@@ -21,25 +16,16 @@ export class BusinessFormStepped {
   @Prop() businessId: string;
   @Prop() testMode: boolean = false;
   @Prop() hideErrors?: boolean = false;
-  @State() isLoading: boolean = false;
+  @State() formLoading: boolean = false;
+  @State() serverError: boolean = false;
   @State() currentStep: number = 0;
   @State() totalSteps: number = 4;
-  @State() serverError: boolean = false;
   @State() errorMessage: string = '';
   @Event() clickEvent: EventEmitter<{ data?: any, name: string }>;
-  @Event() submitted: EventEmitter<{ data: any }>;
-
-
-  private formController: FormController;
-  private api: any;
 
   constructor() {
-    this.sendData = this.sendData.bind(this);
-    this.fetchData = this.fetchData.bind(this);
-  }
-
-  get disabledState() {
-    return this.isLoading;
+    this.incrementSteps = this.incrementSteps.bind(this);
```

Comment:
we can skip binding `this` if we use arrow functions when declaring the function, example: 

line 103.
`incrementSteps = () => {}`

Reply:
Oh I did not know this - thanks for the tip @TiagoFuelber ! 

Updated this!